plugins/github-pull-requests-board/src/api/useGetPullRequestDetails.ts

```diff
@@ -57,6 +57,15 @@ export const useGetPullRequestDetails = () => {
                     state
                   }
                 }
+                commits(last: 1) {
+                  nodes {
+                    commit {
+                      statusCheckRollup {
+                        state
```

Comment:
Does this always return a status? Can this be undefined at any point?

Reply:
Each commit always returns a status, which can be one of the following: ERROR, EXPECTED, FAILURE, PENDING, or SUCCESS.